### PR TITLE
add a section to compare API call volume

### PR DESCRIPTION
**Pull request by mengqiy**
*Issue #, if available:*

*Description of changes:*

Added a section in workload.md to teach user to compare API call volume

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: content/scalability/docs/workloads.md [hunk 1]
```diff
@@ -97,3 +97,29 @@ request:
...
 
 This defragmentation may cause pauses in etcd which could have other affects on the Kubernetes API and controllers. You should avoid frequent modification of large resources or modifying hundreds of resources in quick succession.
+
+## Compare the workloads across 2 clusters
+
+If you have 2 clusters that you think they should have similar performance, but they don't. You should compare the metrics to
+figure out the difference.
+
+A very common question is why the clusters have different latency. It is usually caused by the different volumes of API
+requests. You can run the following CloudWatch LogInsight query to understand the difference.
```
Comment (hakuna-matatah) [inline content/scalability/docs/workloads.md:107]: Nit:
> It is usually caused by the different volumes of API
requests.

Wondering if you want to phrase it this way ? : " It is usually caused by difference in the volume of API requests."

**Comment by hakuna-matatah** [comment]
minor comment ^^^ otherwise LGTM. 
Thanks for adding this David.

File: content/scalability/docs/workloads.md (hunk 1 above)
Comment (mengqiy) [reply content/scalability/docs/workloads.md]: Updated

**Comment by mengqiy** [comment]
@hakuna-matatah  @svennam92 PTAL

**Review by hakuna-matatah — APPROVED** [review]
/lgtm

**Comment by mengqiy** [comment]
@svennam92 PTAL

**Review by svennam92 — APPROVED** [review]

**Comment by svennam92** [comment]
Thank you for the PR, LGTM!

**Pull request merged**